include/pmacc/communication/manager_common.hpp

```diff
@@ -36,3 +36,5 @@ enum {
 };
 
 #define MPI_CHECK(cmd) {int error = cmd; if(error!=MPI_SUCCESS){std::cerr << "<" << __FILE__ << ">:" << __LINE__; throw std::runtime_error(std::string("[MPI] Error"));}}
+
+#define MPI_CHECK_NO_EXCEP(cmd) {int error = cmd; if(error!=MPI_SUCCESS){std::cerr << "[MPI] Error code " << error << " in <" << __FILE__ << ">:" << __LINE__;}}
```

Comment:
To me `NO_EXCEP `does not seem a natural way to shorten. Why not `NO_EXCEPT ` or `NO_THROW` (or same without an underscore), since there are `noexcept `and `nothrow `in C++ ?

Unrelated to this PR, to me it is also more logical to have smth like a non-throwing `MPI_CHECK ` and a throwing `MPI_CHECK_THROW` or `MPI_CHECK_EXCEPT`, but that's a matter of personal opinion I guess.

Reply:
I agree, just used the same naming as we use in `CUDA_CHECK_` macros.
Of course we can refactor both.

Reply:
please change it to `MPI_CHECK_NO_EXCEPT`

Reply:
Ok, but I will change `CUDA_CHECK_NO_EXCEPT` at the same time :)